group-aws/integration.md

```diff
@@ -0,0 +1,34 @@
+# Integration Testing
+
+There are very few integration tests for EC2. Those that
+exist are in test/integration/targets
+
+## Expected test criteria
+
+* Resource creation
+* Resource creation again
+* Resource creation under check mode
+* Resource modification
+* Resource deletion
+
+When adding new features, those features should have tests.
+
+# Github PRs
+There are some PRs with tests:
```

Comment:
You can search for the `test_pull_requests` and `aws` labels, which today returns 32 PRs
https://github.com/ansible/ansible/pulls?utf8=✓&q=is%3Apr%20is%3Aopen%20label%3Aaws%20label%3Atest_pull_requests%20


Ignoring new modules  takes that down to 23
https://github.com/ansible/ansible/pulls?utf8=✓&q=is%3Apr%20is%3Aopen%20label%3Aaws%20label%3Atest_pull_requests%20-label%3Anew_module%20-label%3Anew_plugin

You could filter further by ignoring feature_pull_requests.